theano/gpuarray/tests/test_abstractconv.py

```diff
@@ -196,7 +196,7 @@ def setup_class(cls):
         cls.shared = staticmethod(gpuarray_shared_constructor)
         cls.mode = mode_with_gpu.excluding('cudnn')
 
-    def tcase(self, i, f, s, b, flip, provide_shape, fd=(1, 1, 1)):
+    def tcase(self, i, f, s, b, flip, provide_shape, fd=(1, 1, 1), u=False):
```

Comment:
SkipTest for those cases as well. (You shouldn't just ignore values you don't like, since that would suggest that the test is run and passes.)